Description: V4L1 support was removed in the most recent kernels.
 Update #include statements to rely on v4l-utils's headers.
Origin: Ubuntu
Bug-Ubuntu: https://launchpad.net/bugs/749079
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=621953 
---
 src/v4l.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- camorama.orig/src/v4l.h
+++ camorama/src/v4l.h
@@ -13,7 +13,7 @@
 #include <fcntl.h>
 #include <gtk/gtk.h>
 #include <linux/types.h>
-#include <linux/videodev.h>
+#include <libv4l1-videodev.h>
 #include <signal.h>
 #include <png.h>
 #include <glade/glade.h>
